src/Seld/JsonLint/ParsingException.php

```diff
@@ -15,12 +15,19 @@ class ParsingException extends \Exception
 {
     protected $details;
 
+    /**
+     * @param string $message
+     * @psalm-param array{text?: string, token?: string, line?: int, loc?: array, expected?: array} $details
```

Comment:
`loc` might be documented with more details. Here is its structure: https://github.com/Seldaek/jsonlint/blob/ff2aa5420bfbc296cf6a0bc785fa5b35736de7c1/src/Seld/JsonLint/Lexer.php#L77

Reply:
and `expected` is an `string[]`

Reply:
I will do a followup PR and also add phpstan, so we can verify the changes in the repo here.

Reply:
IIRC, PHPStan will also read the `psalm-param` annotation (it will just be less strict in the validation when it cannot understand the type)

Reply:
> IIRC, PHPStan will also read the `psalm-param` annotation (it will just be less strict in the validation when it cannot understand the type)

yep correct.

with "and also add phpstan" I was refering to add a github action phpstan check in this repo here.

Reply:
https://github.com/Seldaek/jsonlint/pull/71